src/patients/new/NewPatient.tsx

```diff
@@ -10,6 +10,7 @@ import Patient from '../../shared/model/Patient'
 import { RootState } from '../../shared/store'
 import GeneralInformation from '../GeneralInformation'
 import { createPatient } from '../patient-slice'
+import DuplicateNewPatientModal from './DuplicateNewPatientModal'
```

Comment:
There should be some tests to test this new workflow. 

Basically, have an already existing patient in the redux store, try to create a new patient with the same information, and test to make sure that the new modal shows.